### ISSUE TITLE
Ephemeris defaults to DE440

### ISSUE THREAD

**Issue by vhaasteren** [bencebecsy/pta_replicator#3]
When reading a par file with `load_pulsar`, there is an option to set ephem which defaults to DE440. This is problematic when the EPHEM is set in the par file already, because then the wrong ephemeris is used.

Physically one could argue that using the latest ephemeris is always best, especially when simulating data, but this default overwriting the value in the par file violates the principle of least surprise.

**Comment by bencebecsy** [comment]
Yes, that's a good point. The two places I see where we actually use `ephem` is `pint.toa.get_TOAs` when reading in the pulsar and for setting up and `enterprise` `Pulsar` object for saving it. Are those smart enough that if we call them without an `ephem` specified they will just use what's in the par file?

Maybe the default should be `None` in which case we could call these without `ephem` specified and print out a note that we are using whatever's in the par file. But if the user specifies a specific `ephem` we can use that.
